### DISCUSSION TITLE
Bug: npm run generate invalid token

### DISCUSSION THREAD

**Discussion by leecjaster** [wpengine/faustjs#1655] [General]
Several of the listed fixes did not work for me here on a similar issue: https://github.com/wpengine/faustjs/issues/576

## Environment
- `@faustwp/core` version: 0.2.2
- `@faustwp/cli` version: 0.2.13
- `@faustwp/blocks` version: N/A
- `@faustwp/block-editor-utils` version: N/A
- WordPress version: 6.3.2
- Browser, OS, and other relevant software versions: N/A

## Description
When running `npm run generate` locally, I receive the following error below. The URL of `https://hdhypz129th2qx4j654qqmbh2.js.wpenginepowered.com/graphql` is the front-end Atlas staging URL from Wordpress Engine. Is that the correct URL it should be looking at for the graphQL endpoint and updating ACF custom fields in `possibleTypes.json` ?

```
> @faustjs/atlas-blueprint-portfolio@0.2.0 generate
> faust generatePossibleTypes


/Users/lee/sites/REDACTED/node_modules/node-fetch/lib/index.js:273
                                return Body.Promise.reject(new FetchError(`invalid json response body at ${_this2.url} reason: ${err.message}`, 'invalid-json'));
                                                           ^
FetchError: invalid json response body at https://hdhypz129th2qx4j654qqmbh2.js.wpenginepowered.com/graphql reason: Unexpected token < in JSON at position 0
    at /Users/lee/sites/REDACTED/node_modules/node-fetch/lib/index.js:273:32
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async generatePossibleTypes (file:///Users/lee/sites/REDACTED/node_modules/@faustwp/cli/dist/utils/generatePossibleTypes.js:29:18)
    at async file:///Users/lee/sites/REDACTED/node_modules/@faustwp/cli/dist/index.js:52:13 {
  type: 'invalid-json'
```

## Expected vs. Actual Behavior
I anticipate the graphQl endpoint to update the `possibleTypes.json` file.

**Comment by theodesp** [comment]
Hey @leecjaster . You should be using the NEXT_PUBLIC_WORDPRESS_URL env variable in your .env file that points to your  WordPress URL and not your Headless Url since the `/graphql` endpoint is served from WordPress.

See Reference: https://faustjs.org/tutorial/get-started-with-faust#connecting-your-wordpress-site

**Reply by leecjaster** [reply]
@theodesp — Thanks for the help. I do have the Wordpress URL in place in my `.env.local` file. I also saw renaming the file to `.env` was suggested elsewhere, but that did not work either. In the Wordpress admin for the Faust settings, I the Atlas URL in place for the Front-end Site URL field.

Any other ideas? Is there something else locally that could be off?

**Comment by mindctrl** [comment]
@leecjaster I'm going to move this to a Discussion. If Theo's fix above doesn't help, please let us know.
